src/state/queries/post.ts

```diff
@@ -0,0 +1,236 @@
+import {AppBskyFeedDefs} from '@atproto/api'
+import {
+  useQuery,
+  useQueryClient,
+  useMutation,
+  QueryClient,
+} from '@tanstack/react-query'
+import {useSession} from '../session'
+
+export const RQKEY = (postUri: string) => ['post', postUri]
```

Comment:
Would be nice to have this in a typed key generating factory as previously mentioned